internal/controllers/clusterextension_controller.go

```diff
@@ -119,14 +119,25 @@ func checkForUnexpectedFieldChange(a, b ocv1alpha1.ClusterExtension) bool {
 //
 //nolint:unparam
 func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alpha1.ClusterExtension) (ctrl.Result, error) {
+	l := log.FromContext(ctx).WithName("operator-controller")
+
+	// Don't do anything if Paused
+	ext.Status.Paused = ext.Spec.Paused
+	if ext.Spec.Paused {
+		l.Info("resource is paused", "name", ext.GetName(), "namespace", ext.GetNamespace())
+		return ctrl.Result{}, nil
+	}
```

Comment:
I know that in theory we shouldn't do anything, but from a UX perspective I think it might be worth setting some form of status indicating that "no work was done" because it is paused.

What do you think of something like setting `Progressing` to `False` with a reason `Paused`?

Reply:
I'm also OK with this being done in a follow-up since the scope of this PR is to make the two APIs consistent